pennylane/transforms/optimization/merge_rotations.py

```diff
@@ -165,7 +166,9 @@ def qfunc(x, y, z):
                     else:
                         cumulative_angles = fuse_rot_angles(
                             cumulative_angles,
-                            cast_like(stack(next_gate.parameters), cumulative_angles),
+                            cast_like(
+                                stack(next_gate.parameters, like=interface), cumulative_angles
```

Comment:
What is the motivation for this line?

Reply:
This is being done to support `tensorflow` and `pytorch` interfaces.